.github/workflows/release.yml

```diff
@@ -42,6 +42,38 @@ jobs:
           path: ./dist/*.tar.gz
           retention-days: 5
 
+  test-sdist:
+    name: Test sdist
+    needs: [build-sdist]
+    runs-on: ubuntu-latest
+    container:
+      image: "osgeo/gdal:ubuntu-small-3.4.0" # python 3.8.10
+
+    steps:
+      - name: Download sdist from artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: wheelhouse
+
+      - name: Install python
+        run: |
+          apt-get update && apt-get install -y python3 python3-pip
```

Comment:
python is already installed on the docker image, so this step might not be needed? (I noticed because the build log said that numpy was already installed) Although pip might still be needed to install

Reply:
Right!  `python3-pip` came from our existing docker GDAL test, but I went ahead and added `python3` to be safe without thinking it through...